### PR TITLE
 Added  "min" "max" support for inner and outer gauge

### LINKED ISSUE #1: Individual min/max values for gauges
It should be possible to define max and min values for both gauges at once (as is) or for each gauge individualy.

### PR DESCRIPTION
**Pull request by VirtualL**
fixes #1

### CODE REVIEW

**Comment by Rocka84** [comment]
Sorry, I wrote these comments a day ago already but forgot submit them as a review.

File: README.md [hunk 1]
```diff
@@ -53,7 +55,17 @@ colors for then inner gauge are shaded by 25%.
 
 The list is automatically sorted so you don't need to do that in your config - but I recommend it anyways.
 
-## Example
+### min and max config
+
+There is 3 ways to config the minimum and maximum value of the gauge:
```
Comment (Rocka84) [inline README.md]: Actually there are two ways to configure this and a default when you dont configure it at all.

Is suggest something like:
`The min and max values may be configured for both gauges at once or for each gauge individualy and default to 0 to 100.`
as for the color config.

File: dual-gauge-card.js [hunk 2]
```diff
@@ -18,13 +18,20 @@ class DualGaugeCard extends HTMLElement {
...
+    if (!this.config.min & (!this.config.inner.min || !this.config.outer.min)) {
+      this.config.inner.min = this.config.outer.min = 0;
     }
-    if (!this.config.max) {
-      this.config.max = 100;
+    if (!this.config.max & (!this.config.inner.max || !this.config.outer.max)) {
+      this.config.inner.max = this.config.outer.max = 100;
     }
 
+    if (this.config.min) {
```
Comment (Rocka84) [inline dual-gauge-card.js]: It think the individual value should be more important than the global value.
so please use something like `if (!this.config.inner.min) { this.config.inner.min = this.config.min }` here and revert the changes above.

File: dual-gauge-card.js [hunk 3]
```diff
@@ -89,8 +96,13 @@ class DualGaugeCard extends HTMLElement {
     return entity.attributes[attribute];
   }
 
-  _calculateRotation(value) {
-    return (180 - (value - this.config.min) / (this.config.max - this.config.min) * -180) + 'deg';
+  _calculateRotation(gauge,value) {
```
[the remark this answers is not in the document]
Comment (VirtualL) [reply dual-gauge-card.js]: yes , i was using the same in _findColor , great to know :)

File: README.md [hunk 4]
```diff
@@ -48,14 +52,25 @@ If no color is found, the last color in the list is used as a fallback.
...
+
+Colors may be configured for both gauges at once or for each gauge individualy. By default,
+colors for then inner gauge are shaded by 25% (see option _shadeInner_).
 
 The list is automatically sorted so you don't need to do that in your config - but I recommend it anyways.
 
-## Example
+### min and max config
+
+There is 2 ways to config the minimum and maximum value of the gauge:
```
Comment (Rocka84) [inline README.md]: "There are two ways to configure the minimum and maximum values of the gauges"

File: README.md [hunk 5]
```diff
@@ -48,14 +52,25 @@ If no color is found, the last color in the list is used as a fallback.
...
+
+1. **Configuration in the "Config" part** - The vaule for outer and inner gauge will be the same 
+2. **Configuration in outer and inner separately** - outer and inner will have different values , as you set
+    (you must to config the outer **and** the inner gauge, otherwise the the  gauge will have the default values.)
```
Comment (Rocka84) [inline README.md]: "you have to configure" and there's 2 times "the" and an extra space after "otherwise"

And actually, that wouldn't be the case if you would implement my change request to prefer individual config over the global config. If one of the gauges has no own config, it would fallback to the global config.

File: README.md [hunk 6]
```diff
@@ -48,14 +52,25 @@ If no color is found, the last color in the list is used as a fallback.
...
 
 The list is automatically sorted so you don't need to do that in your config - but I recommend it anyways.
 
-## Example
+### min and max config
+
+There is 2 ways to config the minimum and maximum value of the gauge:
+
+1. **Configuration in the "Config" part** - The vaule for outer and inner gauge will be the same 
+2. **Configuration in outer and inner separately** - outer and inner will have different values , as you set
```
Comment (Rocka84) [inline README.md]: drop " , as you set"

**Comment by Rocka84** [comment]
I'm sorry but you didn't react to my change requests or fix the merge conflicts so I have to decline this pull request.
I really appreciate your intention to contribute but this issue should have been fixed weeks ago and I already had code for this ready to go when you opened this pull request.
